apps/backend/tsconfig.json

```diff
@@ -9,5 +9,10 @@
     {
       "path": "./tsconfig.spec.json"
     }
-  ]
+  ],
+  "compilerOptions": {
```

Comment:
I think it's already in the base tsconfig, if so, you shouldn't redeclare these options here.